infrastructure/template.yaml

```diff
@@ -1188,3 +1188,11 @@ Resources:
     Properties:
       AliasName: !Sub "alias/${AWS::StackName}/${Environment}/SnsKmsKey"
       TargetKeyId: !Ref SnsKmsKey
+
+  SnsKmsKeyARNParameter:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Description: The ARN of the SNS KMS Key
```

Comment:
And ARN to ID in description :)

Reply:
! Do'h I chaned that last min and forgot to update the other bits, great spot. Changing now